Pod/Classes/URBNBodyProvider.h

```diff
@@ -0,0 +1,56 @@
+//
+//  URBNCopyProvider.h
+//  Pods
+//
+//  Created by Ryan Garchinsky on 1/15/15.
+//
+//
+
+/*
+ Notes:
+ - Pinterest activity does not support body text, only images
+*/
+
+#import <UIKit/UIKit.h>
+
+@interface URBNBodyProvider : UIActivityItemProvider
+
+/**
+ This text is used for all body properties that are not set, as well as
+    activitity types that are not handled 
+ */
+@property (nonatomic, strong) NSString *defaultBody;
```

Comment:
We may want to init with a default body so that its a required. Otherwise we are never guaranteed it will be there.